modules/cluster-logging-visualizer-indices.adoc

```diff
@@ -39,8 +39,8 @@ To define index patterns and create visualizations in Kibana:
 
 . Create your Kibana index patterns by clicking *Management* -> *Index Patterns* -> *Create index pattern*:
 
-** Users must manually create index patterns to see logs for their projects. Users should create a new index pattern named *app* and use the `@timestamp` time field to view their container logs. 
+** Each user must manually create index patterns when logging into Kibana the first time in order to see logs for their projects. Users should create a new index pattern named *app* and use the `@timestamp` time field to view their container logs.
```

Comment:
```suggestion
** Each user must manually create index patterns when logging into Kibana the first time to see logs for their projects. Users must create an index pattern named `app` and use the `@timestamp` time field to view their container logs.
```

The only times we use bold styling are for UI elements and emphasis:
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

> ! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as appropriate.